app/middleware/journeyAllowed-middleware.ts

```diff
@@ -0,0 +1,12 @@
+import { NextFunction, Request, Response } from 'express';
+import { paths } from '../paths';
+
+const isJourneyAllowedMiddleware = (req: Request, res: Response, next: NextFunction) => {
+  const pathsCopy = { ...paths[req.session.appeal.appealStatus] } || {};
```

Comment:
on new appeals, appeal status is set as DRAFT and not appealStarted, maybe need to double check that one case

Reply:
we don't have a DRAFT status, just a DRAFT ref for the appeal ref number